lib/RemoteAST/RemoteAST.cpp

```diff
@@ -327,7 +327,22 @@ class RemoteASTContextImpl {
       return fail<uint64_t>(Failure::TypeHasNoSuchMember, memberName);
 
     // Fast path: element 0 is always at offset 0.
-    if (targetIndex == 0) return uint64_t(0);
+    if (targetIndex == 0)
+      return uint64_t(0);
+
+    // Only the runtime metadata knows the offsets of resilient members.
```

Comment:
Does the below code not already return failure in this case? It should be fixed more directly. This check here is also incomplete because it doesn’t handle resilient types that might be nested recursively. A full implementation would duplicate type lowering so it’s best to just ask IRGen directly if the type is fixed size or not.